scripts/postinstall.sh

```diff
@@ -34,11 +34,17 @@ fi
 
 echo "${INFO} Injecting waterline...";
 
-pushd "${MOD_DIR}/sails" > /dev/null;
-npm remove waterline;
-npm install "${MOD_DIR}/sails-mysql-transactions/waterline";
+# Replaces the default sails waterline with the Postman edition in the current directory context
+replace_waterline() {
+    npm remove waterline;
+    npm install "${MOD_DIR}/sails-mysql-transactions/waterline";
```

Comment:
This still installs waterline inside SMT on NPM3. Get rid of these fancy functions and simply do repeated if else.

Also the pushd and popd are not balanced.